src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js

```diff
@@ -104,14 +123,23 @@ const InstructorAddCourseCards = props => {
           onChange={handleChange}
         >
           <Form.Item>
-            <label for="courseType">Course Type: </label>
+            <label for="courseType">Program Type: </label>
             <Select placeholder="Select a course" name="course_type">
-              {courses.map(course => (
+              {programs.map(course => (
                 <Option value={course}>{course}</Option>
```

Comment:
```suggestion
                <Option value={program}>{program}</Option>
```

Reply:
the renaming is being done in a different branch feature/styling-instructor-add-course